Sanghyun/app.js

```diff
@@ -37,17 +28,25 @@ appDataSource.initialize()
 
     })
     
+app = express() //express() 함수를 호출하고, app이라는 변수안에 담는 행위 
```

Comment:
```suggestion
const app = express() 
```
app이라는 변수는 const 키워드를 통해 선언해주세요.